includes/compat/trait-wc-stripe-subscriptions.php

```diff
@@ -592,6 +592,10 @@ public function add_subscription_information_to_intent( $request, $order, $prepa
 			$sub_amount += WC_Stripe_Helper::get_stripe_amount( $sub->get_total() );
 		}
 
+		if ( 0 === $sub_amount ) {
```

Comment:
(🤏) May be good to add a comment here about why we're skipping the processing below in this case? For future reference.